Description: Upstream changes introduced in version 0.0.1+svn20110409.0930-3yavdr1~natty
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 eventlircd (0.0.1+svn20110409.0930-3yavdr1~natty) natty; urgency=low
 .
   * patched evmap 03_147a_e02d
 .
 The person named in the Author field signed this changelog entry.
Author: Gerald Dachs <gda@dachsweb.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- eventlircd-0.0.1+svn20110409.0930.orig/etc/03_147a_e02d.evmap
+++ eventlircd-0.0.1+svn20110409.0930/etc/03_147a_e02d.evmap
@@ -28,3 +28,4 @@
  KEY_T     = KEY_TEXT
  KEY_P     = KEY_PAUSE
  KEY_V     = KEY_ZOOM       # EXT
+ KEY_POWER = KEY_POWER2
